datastore/longhorn.go

```diff
@@ -99,15 +99,33 @@ func (s *DataStore) filterCustomizedDefaultSettings(customizedDefaultSettings ma
 	availableCustomizedDefaultSettings := make(map[string]string)
 
 	for name, value := range customizedDefaultSettings {
-		if err := s.ValidateSetting(string(name), value); err == nil {
-			availableCustomizedDefaultSettings[name] = value
-		} else {
-			logrus.WithError(err).Errorf("invalid customized default setting %v with value %v, will continue applying other customized settings", name, value)
+		if s.isSettingValueChanged(types.SettingName(name), value) {
+			if err := s.ValidateSetting(string(name), value); err == nil {
+				availableCustomizedDefaultSettings[name] = value
+			} else {
+				logrus.WithError(err).Warnf("invalid customized default setting %v with value %v, will continue applying other customized settings", name, value)
```

Comment:
Any reason we are changing from error to warning?

Reply:
It's because this message won't be returned to handle but recognized/record instead. 

So change the log level to warning.